adapters_test.go

```diff
@@ -0,0 +1,355 @@
+package monomer_test
+
+import (
+	"testing"
+
+	bfttypes "github.com/cometbft/cometbft/types"
+	codectypes "github.com/cosmos/cosmos-sdk/codec/types"
+	sdktx "github.com/cosmos/cosmos-sdk/types/tx"
+	"github.com/ethereum/go-ethereum/common/hexutil"
+	ethtypes "github.com/ethereum/go-ethereum/core/types"
+	"github.com/kataras/iris/v12/x/errors"
+	"github.com/polymerdao/monomer"
+	"github.com/polymerdao/monomer/testutils"
+	rolluptypes "github.com/polymerdao/monomer/x/rollup/types"
+	"github.com/stretchr/testify/require"
+)
+
+func TestAdaptPayloadTxsToCosmosTxs(t *testing.T) {
+	t.Run("nil tx", func(t *testing.T) {
+		txs, err := monomer.AdaptPayloadTxsToCosmosTxs(nil, nil, "")
+		require.NoError(t, err)
+		require.Empty(t, txs)
+	})
+
+	t.Run("0 txd", func(t *testing.T) {
+		txs, err := monomer.AdaptPayloadTxsToCosmosTxs([]hexutil.Bytes{}, nil, "")
+		require.NoError(t, err)
+		require.Empty(t, txs)
+	})
+
+	simpleSigner := func(_ *sdktx.Tx) error {
+		return nil
+	}
+
+	tests := []struct {
+		name, from            string
+		depTxNum, nonDepTxNum int
+		signTx                monomer.TxSigner
+	}{
+		{
+			name:     "1 dep tx",
+			depTxNum: 1,
+		},
+		{
+			name:        "1 + 1 txs",
```

Comment:
can you please modify the names here to be a bit more descriptive? e.g., `1 deposit, 1 non-deposit`

Reply:
[Rename test cases](https://github.com/polymerdao/monomer/pull/157/commits/f9002d3f3e6e3696fa38f039af9a137318009ee2)